scripts/build

```diff
@@ -6,7 +6,6 @@ require('./util/work-from-root')();
 
 var fsExtra = require('fs-extra');
 var buildProject = require('./util/build-project');
-
```

Comment:
The first set are dependencies with `require`. The second bit is vars. I think they make sense to keep separated? Thoughts?